gluten-core/src/main/scala/io/glutenproject/expression/HashExpressionTransformer.scala

```diff
@@ -30,6 +30,15 @@ class HashExpressionTransformer(
   extends ExpressionTransformer with Logging {
 
   override def doTransform(args: java.lang.Object): ExpressionNode = {
+    original match {
+      case XxHash64(_, seed) if seed != 42 =>
```

Comment:
@marin-ma, can we move the check logic to velox side? See `SubstraitToVeloxPlanValidator.cpp`.
I am afraid this check is not applicable to CH backend. And we prefer to letting each backend do their specific check. Thanks!

Reply:
As of now, we can only get the `seed` value on Java side, because in this case the `seed` is not in the spark2substrait conversion.